http4s/src/main/scala/clue/http4s/Http4sWSBackend.scala

```diff
@@ -48,39 +46,34 @@ final class Http4sJDKWSBackend[F[_]: Async](client: WSClient[F]) extends WebSock
                       s"Unexpected clean close for WS without close frame. URI: [$uri]"
                     )
                   )
-                handler.onClose(connectionId, event) >> release
+                handler.onClose(connectionId, event) >> release // TODO
               }
-            case ExitCase.Errored(t) => handler.onClose(connectionId, t.asLeft) >> release
+            case ExitCase.Errored(t) => handler.onClose(connectionId, t.asLeft) >> release // TODO
             case ExitCase.Canceled   =>
-              connection.sendClose() >>
                 handler.onClose(connectionId,
                                 new GraphQLException(s"WS listener canceled. URI: [$uri]").asLeft
-                ) >> release
+                ) >> release // TODO
           }
           .compile
           .drain
           .start
-          .as(new Http4sJDKWSConnection(connection))
+          .as(new Http4sWSConnection(connection))
       }
 }
 
-object Http4sJDKWSBackend {
-  def apply[F[_]: Async]: Resource[F, Http4sJDKWSBackend[F]] =
-    JdkWSClient.simple[F].map(new Http4sJDKWSBackend(_))
-
-  def fromHttpClient[F[_]: Async](client: HttpClient): Resource[F, Http4sJDKWSBackend[F]] =
-    JdkWSClient[F](client).map(new Http4sJDKWSBackend(_))
+object Http4sWSBackend {
+  def apply[F[_]: Concurrent](client: WSClient[F]): Http4sWSBackend[F] =
+    new Http4sWSBackend(client)
 }
 
-final class Http4sJDKWSConnection[F[_] /*: Sync: Logger*/ ](
+final class Http4sWSConnection[F[_]: Concurrent /*: Sync: Logger*/ ](
   private val conn: WSConnectionHighLevel[F]
 ) extends WebSocketConnection[F] {
   override def send(msg: StreamingMessage.FromClient): F[Unit] =
     conn.send(WSFrame.Text(msg.asJson.toString))
 
   // In high-level WS, we cannot specify close code.
   override def closeInternal(closeParameters: Option[WebSocketCloseParams]): F[Unit] = {
-    val params = closeParameters.getOrElse(WebSocketCloseParams())
-    conn.sendClose(params.reason.orEmpty)
+    Concurrent[F].unit // TODO
```

Comment:
In the new `WSConnectionHighLevel` interface, there is no way to send a close frame. To do so would require using the low-level interface.